src/main/resources/swagger.api/schemas/capability/capabilitiesUpdateRequest.json

```diff
@@ -13,7 +12,14 @@
         "description": "Capability identifier",
         "format": "uuid"
       }
+    },
+    "capabilityNames": {
```

Comment:
Is there any max count, say 100 capability names per request or it is unbound?

Reply:
There is not bound for it